cmd/kubeadm/app/util/config/testdata/defaulting/node/defaulted.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubeadm.k8s.io/v1beta2
+apiVersion: kubeadm.k8s.io/v1beta3
```

Comment:
If I'm not wrong this test should reflect the minimal configuration generated by kubeadm, and considering everything now is optional, I was expecting the content of this file to be further reduced

Reply:
Unfortunately, that's mostly dynamically defaulted stuff there, that makes the test dependent on the environment. We can make `incomplete.yaml` even smaller, but then we have to run the test in a container (controller environment).

Reply:
ok, np
the important point is that the user-provided confige should not have defaults